### PR TITLE
Fix arithmetics in tools/gce-quota-sync

### PR DESCRIPTION
**Pull request by miklosn**
Fix division to use floating point arithmetics (it results in 0.0 otherwise).  Also add sending in the usage value.

### CODE REVIEW

File: tools/gce-quota-sync/gce-quota-sync.py [hunk 1]
```diff
@@ -132,15 +132,15 @@ def _quota_to_series(project, region, quota):
     region: set in converted time series labels
     quota: quota object received from the GCE API
   """
-  labels = dict((k, str(v)) for k, v in quota.items() if k != 'usage')
+  labels = dict((k, str(v)) for k, v in quota.items())
```
Comment (jaketf) [inline tools/gce-quota-sync/gce-quota-sync.py]: why was usage originally omitted / now included ?

File: tools/gce-quota-sync/gce-quota-sync.py [hunk 2]
```diff
@@ -132,15 +132,15 @@ def _quota_to_series(project, region, quota):
...
+
   labels['project'] = project
   labels['region'] = region
   try:
-    value = quota['usage'] / quota['limit']
+    value = quota['usage'] / float(quota['limit'])
```
Comment (jaketf) [inline tools/gce-quota-sync/gce-quota-sync.py]: IIUC python3 this [casting as float is unecessary](https://www.python.org/dev/peps/pep-0238/) when using division operator.

File: tools/gce-quota-sync/gce-quota-sync.py (hunk 1 above)
Comment (miklosn) [reply tools/gce-quota-sync/gce-quota-sync.py]: I have no idea. It was included in the examples as a feature and i definitely found the number to be useful in stackdriver.

File: tools/gce-quota-sync/gce-quota-sync.py (hunk 2 above)
Comment (miklosn) [reply tools/gce-quota-sync/gce-quota-sync.py]: Exactly :) My client ran it on python2, figured it wouldn't hurt.

Comment (jaketf) [reply tools/gce-quota-sync/gce-quota-sync.py]: This is fine for backwards compatibility but your client should get w/ the times :)

Comment (miklosn) [reply tools/gce-quota-sync/gce-quota-sync.py]: Haha yes.

**Pull request merged**